help/destinations/catalog/email-marketing/mailchimp-interest-categories.md

```diff
@@ -0,0 +1,239 @@
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select attributes]** category and select from the list of attributes populated from the [!DNL Mailchimp] API. *Note any custom attributes that have been added to the selected [!DNL Mailchimp] Audience will also be available.*
+
+    The default mappings available between your XDM profile schema and [!DNL Mailchimp Interest Categories] will be as below:
+    | Source Field | Target Field | Notes |
+    | --- | --- | --- |
+    |`IdentityMap: Email`|`Identity: email`| Mandatory: Yes |
+    |`xdm: person.name.firstName`|`Attribute: FNAME`| |
+    |`xdm: person.name.lastName`|`Attribute: LNAME`| |
+    |`xdm: person.birthDayAndMonth`|`Attribute: BIRTHDAY`| |
+
+    Additionally, `ADDRESS` is a special target field known as a `merge field` within your [!DNL Mailchimp] audience. The [[!DNL Mailchimp] documentation](https://mailchimp.com/developer/marketing/docs/merge-fields/) defines the required keys as `addr1`, `city`, `state`, and `zip`, and the optional keys `addr2` and `country`. The values for these fields must be strings. If any of the `ADDRESS` field mappings are present, the destination passes the `ADDRESS` object to the [!DNL Mailchimp] API for update. Any `ADDRESS` fields that are not mapped has its value default to `NULL` except for country which defaults to `US`. 
+    
+    The mappings available for the `ADDRESS` field are as below:
+
+    | Source Field | Target Field |
+    | --- | --- |
+    | `xdm: workAddress.street1` | `Attribute: ADDRESS.addr1` |
+    | `xdm: workAddress.street2` | `Attribute: ADDRESS.addr2` |
+    | `xdm: workAddress.city` | `Attribute: ADDRESS.city` |
+    | `xdm: workAddress.state` | `Attribute: ADDRESS.state` |
+    | `xdm: workAddress.postalCode` | `Attribute: ADDRESS.zip` |
+    | `xdm: workAddress.country` | `Attribute: ADDRESS.country` |
+
+    For example, you want to update the value for `country` with the contact's existing address field `addr1`, `city`, `state`, and `zip` values as `132, My Street, Kingston`, `New York`, `New York` and `12401`. To update the `country` you need to pass the existing values with changes *(if any)* and the new value for country. So the values in your dataset should be `132, My Street, Kingston`, `New York`, `New York`, `12401` and `US`. To reiterate, if you only pass `country` and do not provide values for `addr1`, `city`, `state`, and `zip` they will be overwritten by `NULL`.
+
+    An example with the completed mappings is shown below:
+    ![Platform UI screenshot example showing field mappings.](../../assets/catalog/email-marketing/mailchimp-interest-categories/mappings.png)
```

Comment:
looks like in the screenshot the target field value for first name is incorrect and should be FNAME